tailwind.config.js

```diff
@@ -56,18 +56,14 @@ module.exports = {
     'padding',
     'pointerEvents',
     'position',
+    // TODO [^3]: add preflight (https://tailwindcss.com/docs/preflight)
     'margin',
     'maxWidth',
-    'opacity',
```

Comment:
we're still using opacity for the grid overlay. though it appears to be working in the deployment still 🤔